climada_petals/__init__.py

```diff
@@ -40,22 +35,11 @@
 }
 
 
-def setup_climada_data(reload=False):
-
-    for dirpath in [DEMO_DIR, SYSTEM_DIR]:
-        dirpath.mkdir(parents=True, exist_ok=True)
-
+def copy_repo_data(reload=False):
     for src_dir, path_list in REPO_DATA.items():
         for path in path_list:
             if not path.exists() or reload:
                 src = Path(__file__).parent.parent.joinpath(src_dir, path.name)
                 copyfile(src, path)
 
-
-def init():
-    climada.init()
-    setup_climada_data()
-    setup_logging(CONFIG.log_level.str())
```

Comment:
`climada.init` and `setup_logging` are already called in the `climada.__init__` submodule. I don't see any need to rerun those here.